src/internal/connector/graph_connector_onedrive_test.go

```diff
@@ -237,7 +233,7 @@ func (c *onedriveCollection) withPermissions(
 ) *onedriveCollection {
 	// These versions didn't store permissions for the folder or didn't store them
 	// in the folder's collection.
-	if c.backupVersion < 3 {
+	if c.backupVersion < 4 {
```

Comment:
We have to have some coordination with https://github.com/alcionai/corso/pull/2604 on this one